beacon-chain/blockchain/kzg/kzg.go

```diff
@@ -108,39 +109,48 @@ func VerifyCellKZGProofBatch(commitmentsBytes []Bytes48, cellIndices []uint64, c
 	return ckzg4844.VerifyCellKZGProofBatch(commitmentsBytes, cellIndices, ckzgCells, proofsBytes)
 }
 
-// RecoverCellsAndKZGProofs recovers the complete cells and KZG proofs from a given set of cell indices and partial cells.
+// RecoverCells recovers the complete cells from a given set of cell indices and partial cells.
 // Note: `len(cellIndices)` must be equal to `len(partialCells)` and `cellIndices` must be sorted in ascending order.
-func RecoverCellsAndKZGProofs(cellIndices []uint64, partialCells []Cell) (CellsAndProofs, error) {
+func RecoverCells(cellIndices []uint64, partialCells []Cell) ([]Cell, error) {
 	// Convert `Cell` type to `ckzg4844.Cell`
 	ckzgPartialCells := make([]ckzg4844.Cell, len(partialCells))
 	for i := range partialCells {
 		ckzgPartialCells[i] = ckzg4844.Cell(partialCells[i])
 	}
```

Comment:
Nit:
In this case I prefer to use:
```go
	ckzgPartialCells := make([]ckzg4844.Cell, 0, len(partialCells))
	for _, cell := range partialCells {
        ckzgPartialCells = append(ckzgPartialCells, cell)
	}
```

Both are equivalent and I prefer the "append" version because we don't use the index.
However I don't have a strong opinion on it, but I think we should have a consistent use of one method or the other in the codebase for consistency.